Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageLayoutOptions.swift

```diff
@@ -4,68 +4,95 @@
 
     /// If set the message content will be wrapped into a bubble.
-    static let bubble = Self(rawValue: 1 << 1)
+    static let bubble: Self = "bubble"
 
     /// If set the message bubble will not have a `tail` (rendered by default as a non rounded corner)
-    static let continuousBubble = Self(rawValue: 1 << 2)
+    static let continuousBubble: Self = "continuousBubble"
 
     /// If set the message content will have an offset (from the `trailing` edge if `flipped` is set otherwise from `leading`)
     /// equal to the avatar size.
-    static let avatarSizePadding = Self(rawValue: 1 << 3)
+    static let avatarSizePadding: Self = "avatarSizePadding"
 
     /// If set the message author avatar will be shown.
-    static let avatar = Self(rawValue: 1 << 4)
+    static let avatar: Self = "avatar"
 
     /// If set the message timestamp will be shown.
-    static let timestamp = Self(rawValue: 1 << 5)
+    static let timestamp: Self = "timestamp"
 
     /// If set the message author name will be shown in metadata.
-    static let authorName = Self(rawValue: 1 << 6)
+    static let authorName: Self = "authorName"
 
     /// If set the message text content will be shown.
-    static let text = Self(rawValue: 1 << 7)
+    static let text: Self = "text"
 
     /// If set the message quoted by the current message will be shown.
-    static let quotedMessage = Self(rawValue: 1 << 8)
+    static let quotedMessage: Self = "quotedMessage"
 
     /// If set the message thread replies information will be shown.
-    static let threadInfo = Self(rawValue: 1 << 9)
+    static let threadInfo: Self = "threadInfo"
 
     /// If set the error indicator will be shown.
-    static let errorIndicator = Self(rawValue: 1 << 10)
+    static let errorIndicator: Self = "errorIndicator"
 
     /// If set the reactions added to the message will be shown.
-    static let reactions = Self(rawValue: 1 << 11)
+    static let reactions: Self = "reactions"
 
     /// If set the indicator saying that the message is visible for current user only will be shown.
-    static let onlyVisibleForYouIndicator = Self(rawValue: 1 << 12)
+    static let onlyVisibleForYouIndicator: Self = "onlyVisibleForYouIndicator"
     
     /// If set all the content will have centered alignment. By default, the system messages are centered.
     ///
     /// `flipped` and `centered` are mutually exclusive. Only one of these two should be used at a time.
     /// If both are specified in the options, `centered` is prioritized
-    static let centered = Self(rawValue: 1 << 13)
+    static let centered: Self = "centered"
 }
 
-extension ChatMessageLayoutOptions: CustomStringConvertible {
+extension ChatMessageLayoutOption: CustomStringConvertible {
     /// Returns all options the current option set consists of separated by `-` character.
     public var description: String {
         Self.singleOptions
-            .compactMap { contains($0) ? $0.optionName : nil }
```

Comment:
@evsaev Didn't quite understand this code, what was this doing before? 🤔 Why was the contains needed?